tests/test_mongo.py

```diff
@@ -23,3 +24,14 @@ def store(self, db_name):
             pytest.skip('could not connect to mongodb')
         yield MongoStore(conn[db_name], 'simplekv-tests')
         conn.drop_database(db_name)
+
+    @pytest.fixture()
+    def copy_move_store(self, db_name):
+        class CopyMoveStore(MongoStore, CopyMoveMixin):
+            pass
+        try:
+            conn = pymongo.MongoClient()
+        except pymongo.errors.ConnectionFailure:
+            pytest.skip('could not connect to mongodb')
+        yield CopyMoveStore(conn[db_name], 'simplekv-tests')
```

Comment:
If the test throws an exception, the cleanup method below will never be called (as `yield` will not prevent the exception from bubbling up). 

It looks like other test helpers are structured similarly. Is this intentional or an accident?